### ISSUE TITLE
[Accessibility] - The selected state for the toolbar buttons is not announced by screen reader

### ISSUE THREAD

**Issue by sryarama** [ckeditor/ckeditor4#4543]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Launch ckeditor demo url https://ckeditor.com/ckeditor-4/demo/
2. Launch any screen reader like NVDA/JAWS/Voice Over(iOS).
3. Press bold and type some text.
4. Press ALT+F10 to navigate to toolbar using keyboard.
5. Navigate to Bold button and some other button which is not selected e.g. Italic.

### Expected result

For screen reader users proper information should be conveyed when they access toolbar buttons i.e. selected/not.

### Actual result

When screen reader users navigate to the selected item(Bold) and non selected item(Italic), screen reader is not conveying any information like Bold selected etc...

## Other details

* Browser: Chrome, Safari
* OS: Windows, iOS
* CKEditor version: 4

**Comment by sculpt0r** [comment]
Hi @sryarama ,

I can confirm this on Chrome with JAWS & NVDA.

**Comment by sryarama** [comment]
Hi Team, is there any update on this? It is impacting our accessibility compliance.

**Comment by jacekbogdanski** [comment]
Hey @sryarama, so far we have no progress on this issue due to other priorities in CKEditor 4.

If you looking for a way to push this issue forward, you may contact our [support team](https://ckeditor.com/contact/) to see if there are any options to speed things up.

**Comment by CKEditorBot** [comment]
Closed in https://github.com/ckeditor/ckeditor4/pull/5123
